dotCMS/src/main/webapp/WEB-INF/jsp/tag_manager/tag_manager.jsp

```diff
@@ -548,15 +548,15 @@ td {font-size: 100%;}
         <div class="form-horizontal">
             <dl>
                 <dt>
-                    <label for=""><%= LanguageUtil.get(pageContext, "tag") %>:</label>
+                    <label class="required" for=""><%= LanguageUtil.get(pageContext, "tag") %></label>
```

Comment:
you need to specify the `for` attr to **all labels**

Reply:
Noted